coralproject/talk
# Implement Live Updates for Reported Usernames

## What does this PR do?
- Refactor Activity Dot
- Refactor Community Page
- More generic refactorings
- Split Community Page Query into one for `Reported Usernames` and one for `People`
- Implement live updates for `Reported Usernames Queue`
- Implement dangling items
- People search now returns 10 instead of 5 results
- Show notifications when mutations failed
- Prevent multiple runs of the same mutation
- Fix counts not decreasing when approving or rejecting a username
- Correctly handle pagination

- https://www.pivotaltracker.com/story/show/154224318
- https://www.pivotaltracker.com/story/show/154362182
- https://www.pivotaltracker.com/story/show/154445200

## How do I test this PR?

- Check `People` Search now returning 10 instead of 5 results
- Check every feature we have with usernames and the live updates
  - Report username
  - Change username
  - Reject username
  - Accept username
  - Pagination
